### ISSUE TITLE
Another climbing bug

### ISSUE THREAD

**Issue by vvs-** [opentomb/OpenTomb#197]
In *Return to Egypt* Lara can't climb up here in the original engine.
![screenshot from 2015-07-21 20 33 48](https://cloud.githubusercontent.com/assets/5171934/8807901/092f0dc0-2fe8-11e5-9353-c0094614a9e6.png)

**Comment by vvs-** [comment]
Maybe related to #86.

**Comment by Lwmte** [comment]
You mean you was able to grab the slope ledge and climb up? It's an old bug - either there's no check for ledge slope angle or slope angle constant to check with is wrong.

**Comment by vvs-** [comment]
Yes. It's possible to climb it up from water room below. This breaks a puzzle which required very specific way to get out of water.

**Comment by vvs-** [comment]
BTW it breaks the final puzzle in _Atlantean Stronghold_ too. Lara should only get access to upper hole by jumping from the raised platform and not from climbing pyramid from above.
